tesseract_core/runtime/tree_transforms.py

```diff
@@ -165,4 +157,17 @@ def filtered_pos_func(*args):
 
         return path_outputs
```

Comment:
I think @dionhaefner meant for the jax template to use `filtered_pos_func` but this could work too (main advantage is that no modification of jax templates necessary!). Slight refactor below, unfortunately I can't comment on unchanged lines so won't be fully in context.
```suggestion
    def filtered_pos_func(*args):
        new_inputs = dict(zip(input_paths, args))
        return filtered_func(new_inputs)
```

Reply:
you might also only want to define this in the if statement below.

Reply:
maybe also leave defining this until the after the if statement.